### PR TITLE
New PR for validateProperties, which were holding up an earlier PR.

### PR DESCRIPTION
**Pull request by lleonard-msft**
This PR is the new home for the validateProperties topics that were holding up [PR #2504](https://github.com/microsoftgraph/microsoft-graph-docs/pull/2504).

I separated them out so that we could get the other work in PR #2504 in review while we figure out the changes needed to the these topics to validate,

### CODE REVIEW

File: api-reference/beta/api/directoryobject_validateproperties.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  "entityType": "Group",
+  "displayName": "test",
+  "mailNickname": "test",
+  "onBehalfOfUserId": "onBehalfOfUserId-value"
+}
+```
+
+### Response
+```http
+HTTP/1.1 200
```
Comment (dkershaw10) [inline api-reference/beta/api/directoryobject_validateproperties.md]: This needs to be a 422. The rest seems fine.
Try re-adding the MD comments from before.

File: api-reference/beta/api/group_validateproperties.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+{
+  "displayName": "MyPrefix_test_mysuffix",
+  "mailNickname": "MyPrefix_test_mysuffix"
+}
+```
+
+### Response
+```http
+HTTP/1.1 200 OK
```
Comment (dkershaw10) [inline api-reference/beta/api/group_validateproperties.md]: 422
Re-instate the MD scanner comments

**Comment by jasonjoh** [comment]
@lleonard-msft is this still ongoing?

**Comment by dkershaw10** [comment]
@lleonard-msft For now I think this is good.  Please merge this.  We can always update it with additional MD scanner info once we know what it needs to look like.

**Pull request merged**